src/api/projects/queries.js

```diff
@@ -42,6 +42,27 @@ export const MUTATION_UPDATE_PROJECT = `
     }
 `;
 
+// language=GraphQL
+/**
+ * Mutation for generating new integration token
+ */
+export const MUTATION_GENERATE_NEW_INTEGRATION_TOKEN = `
+  mutation generateNewIntegrationToken(
+    $id: ID!
+  ) {
+    generateNewIntegrationToken(id: $id)
+    {
+      record {
+        id
```

Comment:
hard to maintain. User `Project` fragment or return just the `token`. I think, the second is better